c7n/resources/mq.py

```diff
@@ -83,10 +85,58 @@ def get_dimensions(self, resource):
                  'Value': "{}-1".format(resource['BrokerName'])}]
 
 
+@MessageBroker.filter_registry.register('vpc')
+class VpcFilter(VpcFilter):
+    """Filter a resource by its VPC id. Choose to select resources that are in that VPC or select
+    resources that are not in that VPC using the 'op' field. Security group ID is used to get
+    the VPC ID of a message broker, since DescribeMessageBrokers does not return a VPC ID.
```

Comment:
do not include implementation details in the docstrings here, it's not of any use for the end user. This is far too verbose